provisioning/ubuntu-provision.sh

```diff
@@ -257,6 +263,40 @@ function install_vagrant() {
   fi
 }
 
+## Install asdf
+function install_asdf() {
+  reload_shell
+  if ! asdf version >/dev/null 2>&1
+  then
+    local archive install_dir profile_script
+    archive=/tmp/asdf.tgz
+    install_dir=/opt/asdf
+    profile_script=/etc/profile.d/asdf.sh
+
+    curl --fail --silent --show-error --location "https://github.com/asdf-vm/asdf/archive/refs/tags/v${ASDF_VERSION}.tar.gz" --output "${archive}"
```

Comment:
Surprised to see "achive" instead of "releases" here.

Reply:
Downloading the targz archive, which has a different endpoint: the ZIP and tar archives which contains the code are automatically generated by GitHub for *each* tag.
Then, each release associated to this tag has references to these 2 files, but the files are NOT attachments of the release.